src/api/methods.mjs

```diff
@@ -82,40 +103,23 @@ export default [
 	'apps.getScore',
 	'apps.sendRequest',
 
+	/**
+	 * Auth
+	 * Methods for working with authorization
+	 */
+	'auth.checkPhone',
+	'auth.confirm',
+	'auth.restore',
+
 	/**
 	 * Audio
 	 * Methods for working with audio
 	 */
-	'audio.get',
-	'audio.getById',
-	'audio.getLyrics',
-	'audio.search',
 	'audio.getUploadServer',
 	'audio.save',
 	'audio.add',
-	'audio.delete',
 	'audio.edit',
-	'audio.reorder',
-	'audio.restore',
-	'audio.getAlbums',
-	'audio.addAlbum',
-	'audio.editAlbum',
-	'audio.deleteAlbum',
 	'audio.moveToAlbum',
-	'audio.setBroadcast',
-	'audio.getBroadcastList',
-	'audio.getRecommendations',
-	'audio.getPopular',
-	'audio.getCount',
```

Comment:
According to <https://vk.com/dev/audio_api>, these methods were removed.
I left only the methods which pages are not 404.

Reply:
Ok, VK docs are strange as always, but I will fix this.